Typescript_React/src/TodoListItem.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Todo, ToggleTodo } from './Types';
+
+interface TodoListItemProps{
+    todo : Todo;
+    toggleTodo:ToggleTodo
+};
+
+
+export const TodoListItem:React.FC<TodoListItemProps> = ({ todo,toggleTodo }) =>{
+    return <li>
+             <label style={{textDecoration :todo.complete? "line-through" : undefined}}>
+              <input type="checkbox" checked={todo.complete} onChange={() => toggleTodo(todo)}/>
+              {todo.text}
+             </label>
+            </li>;
```

Comment:
you can write this as below

`export const TodoListItem:React.FC<TodoListItemProps> = ({ todo,toggleTodo }) => <li>
             <label style={{textDecoration :todo.complete? "line-through" : undefined}}>
              <input type="checkbox" checked={todo.complete} onChange={() => toggleTodo(todo)}/>
              {todo.text}
             </label>
            </li>`

Reply:
Also no need to explicitly give value undefined. 

` <label style={{textDecoration: todo.complete? "line-through" :  ""}}>`

 
 OR 
 
`  <label style={{textDecoration: todo.complete && "line-through" }}>`